package.json

```diff
@@ -9,7 +9,7 @@
     "react-redux": "^5.0.7",
     "react-scripts-ts": "3.1.0",
     "redux": "^4.0.1",
-    "styled-components": "^4.1.1"
+    "styled-components": "3.4.10"
```

Comment:
I believe there is a problem with type definitions for version `4.x.x` of styled-components. I get the following issue:
```
typescript Namespace 'React' has no exported member 'ComponentPropsWithRef'
```
In order to un-block myself i went a major down to continue the work. Will see if they have fixed the issue after this PR.